docs/scaffolds/bladder.rst

```diff
@@ -0,0 +1,90 @@
+Bladder Scaffold
+================
+
+The current bladder scaffold is ``3D bladder with Urethra 1`` built from ``class MeshType_3d_bladderurethra1``.
+The cat variant is shown in :numref:`fig-scaffoldmaker-cat-bladder`.
+
+.. _fig-scaffoldmaker-cat-bladder:
+
+.. figure:: ../_images/scaffoldmaker_cat_bladder.jpg
+   :align: center
+
+   Cat bladder scaffold.
+
+As with the other organ systems, the lower urinary tract (LUT) contains different sub-scaffolds consisting of the bladder, its ureters, and the urethra.
+
+The bladder scaffold is a 3-D volumetric model, designed topologically as a hollow organ defined by a central line which follows the path of a bladder traced from an image.
+
+The current scaffold consists of different parts to represent the bladder body, bladder neck, ureters opening on the bladder, and the urethra. The neck of the bladder is represented in the scaffold as a narrowing of the body at the region where ureters penetrate the surface into the lumen of the bladder.
+
+.. note::
+
+   Separate scaffold will be provided for the bladder.
+
+Variants
+--------
+
+The bladder scaffold is provided with parameter sets for the following five species, which are very similar in nature but have small, intricate differences:
+
+* Cat
+* Human
+* Mouse
+* Pig
+* Rat
+
+These variants' geometry and annotations are best viewed in the **Scaffold Creator** tool in the ABI Mapping Tools. On the web, the latest published generic bladder scaffold variants can be viewed on the `SPARC Portal <https://sparc.science/>`_ by searching for ``bladder``, filtering for models, selecting a variant and viewing the scaffold in its Gallery tab.
+
+The bladder scaffold script generates the scaffold mesh and geometry based on some parameters include the diameters for different axes, wall thickness, neck angle (for bladder only), etc., which are all defined for the bladder and urethra separately. The ability to control these parameters, as well as variations of them throughout the length of the bladder and urethra allows the scaffold to be configurable for different species and individuals.
+
```

Comment:
Add as separate paragraph: The urethra is optional and can be turned on with the *Include urethra* setting. By default the ureter inlets are marker points on the inner (luminal) surface, but by checking the *Include ureters* option these can be made as 3D element structures.